src/hyppo/freeze.asm

```diff
@@ -790,6 +792,7 @@ freeze_prep_jump_table:
         !16 do_freeze_prep_thumbnail
         !16 do_freeze_prep_viciv
 	!16 do_freeze_prep_none
+	!16 do_freeze_prep_none
```

Comment:
why do we add another pointer to the jump table here?

Reply:
Ah yes, I meant to comment on this.  It should have been added when this commit was made: https://github.com/MEGA65/mega65-core/commit/b88ea0de01d13af47f5757c16ed8cc4c2f410975
Luckily the first entry in `unfreeze_prep_jump_table` just below is a `do_unfreeze_prep_none` so it was working (i.e. not doing anything).  But could be source confusion or bugs later with that missing entry.